src/pe_source/data/pe_db/config.py

```diff
@@ -2,22 +2,20 @@
 
 # Standard Python Libraries
 from configparser import ConfigParser
+import logging
 import os
 
 # Third-Party Libraries
 from importlib_resources import files
 import requests
 import shodan
 
-# cisagov Libraries
-from pe_reports import app
-
 # Configuration
 REPORT_DB_CONFIG = files("pe_reports").joinpath("data/database.ini")
 
 
 # Setup logging to central file
-LOGGER = app.config["LOGGER"]
+LOGGER = logging.getLogger(__name__)
```

Comment:
@DJensen94 
Do we want a separate logging file at this directory? If not, I think we will want to disregard this change.

Reply:
app.config won't pass the tests for some reason, it can't find the app

Reply:
If this is staying as is, then the `Setup logging to central file` comment needs to change.

Reply:
We are calling the Logger this way because importing app has a circular import error. The logs still go to the central logging file, this just avoids the circular import.

Reply:
Can you also please add a comment here similar to https://github.com/cisagov/pe-reports/pull/282/commits/9ea37695b812308973b8dc71a37b72c18ea8fc47?

Reply:
I just noticed that both of those comments (https://github.com/cisagov/pe-reports/commit/9ea37695b812308973b8dc71a37b72c18ea8fc47 and https://github.com/cisagov/pe-reports/pull/282/commits/fce4e512f97528765a5bb0e6fa61b72958845a9e) have a small typo:
`app.config("LOGGER")` should be `app.config["LOGGER"]`

Reply:
@dav3r both typos addressed in the following commit: [88d3b59](https://github.com/cisagov/pe-reports/pull/282/commits/88d3b596817e886af385d6844c05cc10e1de8409)

Reply:
I'm satisfied with the changes @DJensen94 has made here, but I leave it to @cduhn17 (who started this thread) to resolve this conversation if they feel their concerns have been addressed.

Reply:
@dav3r and @DJensen94 , 
I think we have addressed all of the issues. 
Thanks to all!!